.github/workflows/go-driver.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go-version: [ '1.18', '1.19' ]
+        go-version: [ '1.20', '1.21' ]
 
     defaults:
       run:
```

Comment:
The PG13 commit omits the command       run: go test . -v, while the rest mostly have them. Is this intentional? @jrgemignani Otherwise, looks good, but I will hold off on merging for now.

Reply:
@dehowef `run: go test . -v` is at the end of this file. Other PRs(on other branches) just removed an extra line at the end of this file, but this one does not have any.